moderncv-template.tex

```diff
@@ -0,0 +1,243 @@
+	\item Use the \texttt{today} command to show the date when I last modified this document.
+	\item Put the dates on the right hand side, instead of at the beginning of each row.
+	\item Add a contents section. This is recommended for a document with more than 10 pages.
+	\item Add the page number at the bottom right of each page.
+	\item Utilize the \texttt{input} command to add a section directly from another \texttt{.tex} file.\newline{When each section is long, this keeps the main document short and easy to maintain.}
+\end{itemize}}
+
+\vspace{-10pt}
+
+%----------------------------------------------------------------------------------------
+%	DISCLAIMER SECTION
+%----------------------------------------------------------------------------------------
+
+\section{Disclaimer}
+
+\cvitem{$\bullet$}{This template works best for long documents, such as a full CV or a file for all project descriptions. \newline{For industry job applications, it is still recommended to submit a one-page resume and be concise.}}
+
+%----------------------------------------------------------------------------------------
+%	EDUCATION SECTION
+%----------------------------------------------------------------------------------------
+
+\section{Education}
+% Arguments not required can be left empty
+
+\cventry{$\bullet$}{Ph.D. in Subject}{University of Elsewhere}{Country CC}{2016 -- Present. GPA: 4.0/4.0}{}
+\cvitem{}{Research areas: field one, field two, and field three.}
+
+\cventry{$\bullet$}{B.S. in Subject}{University of Somewhere}{Country AB}{2012 -- 2016. GPA: 3.88/4.0}{}
+\cvitem{}{Minor in another subject.}
+
+%\vspace{-10pt}
+
+%----------------------------------------------------------------------------------------
+%	WORK EXPERIENCE SECTION
+%----------------------------------------------------------------------------------------
+
+\section{Work Experience}
+
+\subsection{Research Assistant, Subject, University of Elsewhere}
+
+\cventry{$\bullet$}{Research Project Name}{}{January 2017 -- Present}{}{First line of description.
+\newline{Second line of description.}}
+
+\subsection{Summer Intern, Division X, Company ABCDE}
+
+\cventry{$\bullet$}{Project Name}{}{May 2016 -- August 2016}{}{First line of description.
+\newline{Second line of description.}}
+
+
+\newpage
+
+%----------------------------------------------------------------------------------------
+%	COMPUTER SKILLS SECTION
+%----------------------------------------------------------------------------------------
+
+\section{Computer Skills}
+% Just three lines
+\cvitem{}{Proficient in $\mathsf{R}$, Python, C++}
+\cvitem{}{Familiar with SAS, MATLAB, SQL, Java}
+
+
+%----------------------------------------------------------------------------------------
+%	PUBLICATIONS SECTION
+%----------------------------------------------------------------------------------------
+
+% These are input files because they can get much longer over time.
+% yyyy-mm-dd is the date when I modified the input file.

```

Comment:
I see why you do this and it makes sense if you manage your CV, but I don't think this is something I want in this repo as it only creates file clutter (already rather messy).

Reply:
@stephanlachnit Do you mean that I should remove the additional input files completely, or you just don't want the dates in the filenames?

Reply:
Remove the additional files completely

Reply:
Maybe it would be an idea to have a `contributed` folder in which selected user contributions (requirement: each as one single file) could be given together with a _use at your own risk_ note or similar. That way the community might benefit from tweaks and changes that people have found over the years, but are not „official” enough or too special to get merged.

Reply:
I guess that would work, but I don't really see the point. One comprehensive template should be enough, since when we add new features like in #32 I don't want to update every template (I mean, that is the entire point of a template).

If there are things to improve in the template, sure thing, but adding tons of examples that no one reads anyway doesn't help anyone.

Reply:
@stephanlachnit I appreciate the discussion, and I have removed the input files in this pull request. 
I keep everything else in [my own separate repo](https://github.com/star1327p/moderncv-template).